contracts/internal/EverlongPositions.sol

```diff
@@ -169,17 +174,61 @@ abstract contract EverlongPositions is EverlongBase, IEverlongPositions {
     // │ Position Closing (Internal)                             │
     // ╰─────────────────────────────────────────────────────────╯
 
+    /// @inheritdoc EverlongBase
```

Comment:
Not for this PR, but I think we'll want to re-evaluate our use of `inheritdoc`. If we're going to have documentation for this function, I'd want it to live here since this is where the function is implemented. Generally speaking, I prefer duplicated docs to not having any where the function is actually implemented.